### DISCUSSION TITLE
pls-cli x pls

### DISCUSSION THREAD

**Discussion by joaociocca** [guedesfelipe/pls-cli#85] [Ideas]
I started using both [pls](https://github.com/pls-rs/pls) and [pls-cli](https://guedesfelipe.github.io/pls-cli) recently, which at first borked my system (I then proceeded to rename "this" `pls` in bin to `pls-cli` and everything worked out)... 
It's such a small world lol

**Comment by guedesfelipe** [comment]
It's a giant small world 😂

**Reply by joaociocca** [reply]
the world is a quail egg lol
